pkg/helm/watches/watches.go

```diff
@@ -18,8 +18,9 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
+	"os"
 
-	yaml "gopkg.in/yaml.v2"
+	"github.com/ghodss/yaml"
```

Comment:
Is this a conscious change and why?

Reply:
It is. When `gopkg.in/yaml.v2` unmarshals, the resulting `map[string]interface{}` child maps with type  `map[interface{}]interface{}`. `github.com/ghodss/yaml` gets this right with `map[string]interface{}` all the way down.